README.md

```diff
@@ -1,6 +1,7 @@
 ## Welcome to YETI
 [![Build Status](https://api.travis-ci.org/yeti-switch/yeti-web.svg?branch=master)](https://travis-ci.org/yeti-switch/yeti-web)
 
+[![Coverage Status](https://coveralls.io/repos/github/Ivanov-Anton/yeti-web/badge.svg?branch=master)](https://coveralls.io/github/Ivanov-Anton/yeti-web?branch=master)
```

Comment:
This link should not point to private account https://coveralls.io/repos/github/Ivanov-Anton